targets/tizenrt-artik05x/app/README.md

```diff
@@ -0,0 +1,82 @@
+### About
+
+This directory contains files to run IoT.js on [TizenRT](https://github.com/Samsung/TizenRT).
+
+WARNING: **This document is not 100% accurate since Artik05x board with tooling is not available yet**
+
+### How to build
+
+#### 1. Set up the build environment for Artik05x board
+
+Clone IoT.js and TizenRT into iotjs-tizenrt directory
+
+```bash
+$ mkdir iotjs-tizenrt
+$ cd iotjs-tizenrt
+$ git clone https://github.com/Samsung/iotjs.git
+$ git clone https://github.com/Samsung/TizenRT.git tizenrt
+```
+The following directory structure is created after these commands
+
+```bash
+iotjs-tizenrt
+  + iotjs
+  |  + targets
+  |      + tizenrt-artik05x
+  + tizenrt
+```
+
+#### 2. Add IoT.js as a builtin application for TizenRT
+
+```bash
+$ cd tizenrt/apps/system
+$ mkdir iotjs
+$ cp ../../../iotjs/targets/tizenrt-artik05x/app/* ./iotjs/
```

Comment:
Application files are located in `tizenrt-artik05x/app/iotjs/*`. Please change `app/*` to `app/iotjs/*` or move application files to `app/` folder.